honcho/process.py

```diff
@@ -36,9 +36,16 @@ def __init__(self,
         self._child = None
         self._child_ctor = Popen
 
-    def run(self, events=None, ignore_signals=False):
+    def run(self, events=None, ignore_signals=False, process_kwargs=None):
         self._events = events
-        self._child = self._child_ctor(self.cmd, env=self.env, cwd=self.cwd)
+        process_kwargs = process_kwargs or {}
+        kwargs = dict(
```

Comment:
Does this also override `self.env` and `self.cwd`? I guess it's best to merge env, cwd passed via `process_kwargs`

Reply:
good point, I'll merge `env` instead of overwriting it. This will overwrite `cwd` if it's passed in, which should be the expected behavior, right?